### ISSUE TITLE
False positives

### ISSUE THREAD

**Issue by Coariz** [tiredvekster/LightAntiCheat#6]
```
[20:54:28 INFO]: (Light) GelloJoe was flagged for BadPacketsD (1/10)
[20:54:28 INFO]: (Light) GelloJoe was flagged for BadPacketsD (2/10)
[20:54:29 INFO]: (Light) GelloJoe was flagged for BadPacketsD (3/10)
[20:54:29 INFO]: (Light) GelloJoe was flagged for BadPacketsD (4/10)
[20:54:31 INFO]: (Light) GelloJoe was flagged for BadPacketsD (5/10)
[20:54:32 INFO]: (Light) GelloJoe was flagged for BadPacketsD (6/10)
[20:54:36 INFO]: (Light) GelloJoe was flagged for BadPacketsD (7/10)
[20:54:36 INFO]: (Light) GelloJoe was flagged for BadPacketsD (8/10)
[20:54:37 INFO]: (Light) GelloJoe was flagged for BadPacketsD (9/10)
[20:54:38 INFO]: (Light) GelloJoe was flagged for BadPacketsD (10/10)
[20:54:38 INFO]: (Light) GelloJoe was punished for BadPacketsD
[20:54:38 INFO]: [Essentials] CONSOLE issued server command: /kick GelloJoe BadPacketsD
[20:54:38 INFO]: GelloJoe lost connection: BadPacketsD
[20:54:38 INFO]: [Essentials] Player Console kicked GelloJoe for BadPacketsD.

```

Moreover, I got flagged for criticalsB. I am using a plugin called valhallaMMO and it is flagging its "skills" features

**Comment by tiredvekster** [comment]
Hi! I will try to fix these false positives in the next update! If you notice any other false positives with ValhallaMMO, please let me know when they occur. I don't want to search for them on my own xD

**Comment by Coariz** [comment]
Hello, I'm back again. So I've been tinkering with valhallammo and here's all the false positives that I've found. This also applies to other plugins that modify or "buffs" player movement. I hope next update it is fixed :))
https://pastes.dev/IrSW7IXjNP

**Comment by F3F5** [comment]
@tiredvekster 
12:10:26 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (1/10)
[12:10:26 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (2/10)
[12:10:27 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (3/10)
[12:10:27 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (4/10)
[12:10:28 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (5/10)
[12:10:29 INFO]: (Light) MichaelKozlov was flagged for BadPacketsD (6/10)

**Comment by tiredvekster** [comment]
Hello there! Hopefully, these false positives have been addressed in LAC 1.2.5.
